addons/web/static/src/views/widgets/ribbon/ribbon.scss

```diff
@@ -63,12 +47,3 @@
     }
```

Comment:
```scss
& span {
    left: -15px;
    top: 30px;
    [...]
}
```
These left and top values... 30px, -15px... can we replace them with something less arbitrary ?